composeApp/src/androidMain/kotlin/it/fast4x/rimusic/ui/screens/player/Lyrics.kt

```diff
@@ -337,7 +337,6 @@ fun Lyrics(
                 val result = withContext(Dispatchers.IO) {
                     try {
                         var translation: Translation?
-                        var translation2: Translation?
                         if(destinationLanguage == Language.AUTO){
                             translation = translator.translate(
                                 textToTranslate,
```

Comment:
It is possible to have at most one translation call at once, but this requires a (kind of) complicated change in the "output section". Should I do that?

Reply:
But two translation it's just implemented by @aneesh, what's the difference?

Reply:
In theory we only need one translation call. In the first version we used a second translation for getting the name of the language of the source to get the pronunciation of the source. The 3rd translation call was for getting also a translation. Now we only need two translations (because of a bug with traditional Chinese). If the bug in the translator with incorrect source language gets fixed, then we really only need one translation.

Reply:
My original point was that with the code base of the review, both translations have to be evaluated in the worst case, when with a different code only one is needed in each case (I think it is possible this way).

Reply:
> But two translation it's just implemented by @aneesh, what's the difference?

"aneesh" is not the one from this project.